node-data/src/bls.rs

```diff
@@ -165,9 +165,7 @@ fn read_from_file(
         )
     })?;
 
-    let mut hasher = Sha256::new();
-    hasher.update(pwd.as_bytes());
-    let hashed_pwd = hasher.finalize().to_vec();
+    let hashed_pwd = hash_sha256(pwd);
```

Comment:
```suggestion
    let aes_key = hash_sha256(pwd);
```

While this is a hashed password, it is used as a key in the context of AES, so I would suggest calling it key.